### ISSUE TITLE
Place not being saved in Daily Devotional

### ISSUE THREAD

**Issue by skyblue002** [bibletime/bibletime#200]
The app consistently saves my position if I have tabs open for various versions of the Bible. But, my position in the Daily Devotional is not being properly saved. I tried creating a new session instead of using the default session, by going to `Window > Save as New Session...` Afterwards, my place in the Daily Devotional was saved once or twice, but then got reverted to an earlier place. That is, when I was reading 10.17 and shifted to 10.18. At first, that change was saved. Then, it reverted to 10.17. I'm not sure why this is happening. Are there permissions for certain files that I should check? Or, perhaps a file with my settings got corrupted? If you let me know where to look, I'd be happy to check it out. Thank you for any help you can provide!

Note: I'm using BibleTime 2.11.2 on Manjaro Linux, an Arch derivative

**Comment by gholmlund** [comment]
I have not been able to reproduce this on ubuntu 19.10 or arch. Can you reproduce this by:
1. open BibleTime
2. change the daily reference
3. close BibleTime
   (repeat)

You can see the information saved by this grep:
      grep window ~/.bibletime/bibletimerc  |grep key
      0\window\0\key=04.12
      0\window\1\key=Revelation of John 2:1

**Comment by skyblue002** [comment]
@gholmlund Thank you! I will use that command and track the app with it each time I use it. Right now, it is registering as saving my place in the Daily Devotional. But if/when it changes again, I'll have my eye on it and will reply with the output.

**Comment by skyblue002** [comment]
I encountered a strange issue with BibleTime to do with the Daily Devotional.

I opened the KJVA Bible in a new tab to look up a passage mentioned in the Daily Devotional. Then, after closing that tab, I noticed that the content of the Daily Devotional reverted to 01.01, even though the controls show it at the current date, 10.20:

![Bibletime problem](https://user-images.githubusercontent.com/41849249/67156996-bccc6f80-f2f3-11e9-82a3-d1206aa67cf2.png)

I fed the command into the terminal, and it says this

```
0\window\0\key=Mark 1:1
0\window\1\key=Mark 1:1
0\window\2\key=01.01
0\window\3\key=Exodus 20:0
0\window\4\key=Matthew 6:34
0\window\5\key=Matthew 5:48
0\window\6\key=Matthew 5:46
0\window\7\key=Matthew 7:12
1\window\0\key=Mark 1:1
1\window\1\key=Mark 1:1
1\window\2\key=10.19
```

I had created my own session after initially encountering the issue. That was my attempt to solve it. I would assume that this is what the `1\window...` refers to, as those are the sections currently open.

I immediately closed BibleTime, ran the code in the terminal again, and it shows that it was reverted to 01.01:

```
0\window\0\key=Mark 1:1
0\window\1\key=Mark 1:1
0\window\2\key=01.01
0\window\3\key=Exodus 20:0
0\window\4\key=Matthew 6:34
0\window\5\key=Matthew 5:48
0\window\6\key=Matthew 5:46
0\window\7\key=Matthew 7:12
1\window\0\key=Mark 1:1
1\window\1\key=Mark 1:1
1\window\2\key=01.01
```
When I opened BibleTime again right after that, it went straight to 01.01 in the Daily Devotional.

**Comment by gholmlund** [comment]
qt5ct is a style customization program for Qt. I believe the message is just for your information, not an warming/error.

The corrupt data messages are from Sword and relate to uncompressing the modules. BibleTime has no control over these messages. I don't believe they are the cause of the problem.

I have been testing creating new sessions to reproduce this, but so far I have not seen the problem.

**Comment by skyblue002** [comment]
I appreciate you looking into it. Since you can't reproduce it, just let me know if you need logs or command line output from my machine. Maybe the problem is on my machine, but if it's not, it could help you to fix the issue to prevent this from happening to other users. I know that when I was dealing with it, it was an on/off problem. It didn't happen every time. But, as I said, I'm happy to help. So, if you need further information, just let me know. :smile:

**Comment by gholmlund** [comment]
I have a reproducible case. I have 3 windows open each with a daily module and they are Auto-Tiled. I then switch to tabbed and close BT. The wrong values are written to $HOME/.bibletime/bibletimerc.
The problem occurs in the reference chooser combobox. I am still studying the code to find the right fix.

**Comment by gholmlund** [comment]
I have fixed this in the stable-2.11 and master branches.

**Comment by skyblue002** [comment]
Thank you! 😄
